bin/node/rpc/src/lib.rs

```diff
@@ -127,6 +144,11 @@ pub fn create_full<C, P, M, SC>(
 			BabeRPCHandler::new(client, shared_epoch_changes, keystore, babe_config, select_chain)
 		)
 	);
+	io.extend_with(
+		sc_finality_grandpa_rpc::GrandpaApi::to_delegate(
+			GrandpaRpcHandler::new(shared_voter_state, shared_authority_set)
```

Comment:
Anything preventing this RPC working on light clients? Would it be worth adding there?

Reply:
The light client does not participate in the GRANDPA protocol, currently it does not participate in gossip at all. In the future it might be possible to have the light client listen to just the global GRANDPA gossip topic (rather than rounds) and run the GRANDPA observer instead (the lightweight observer is already implemented). But even in that scenario it won't be running a full voter and therefore we won't need this API.